qdrant-landing/content/articles/qdrant-1.8.x.md

```diff
@@ -0,0 +1,110 @@
+
+This performance increase can have a dramatic effect on hybrid search implementations. [Read more about how to set this up.](https://qdrant.tech/articles/sparse-vectors/)
+
+FYI, sparse vectors were released in [Qdrant v.1.7.0](https://qdrant.tech/articles/qdrant-1.7.x/#sparse-vectors). They are stored using a different index, so first [check out the documentation](https://qdrant.tech/documentation/concepts/indexing/#sparse-vector-index) if you want to try an implementation.
+
+## CPU resource management
+
+Indexing is Qdrant’s most resource-intensive process. Now you can account for this by allocating compute use specifically to indexing. You can assign a number CPU resources towards indexing and leave the rest for search. As a result, indexes will build faster, and search quality will remain unaffected.
+
+This isn't mandatory, as Qdrant is by default tuned to strike the right balance between indexing and search. However, if you wish to define specific CPU usage, you will need to do so from `config.yaml`.
+
+This version introduces a `optimizer_cpu_budget` parameter to control the maximum number of CPUs used for indexing. 
+
+> Read more about `config.yaml` in the [Configuration document](https://qdrant.tech/documentation/guides/configuration/).
+
+```yaml
+    # CPU budget, how many CPUs (threads) to allocate for an optimization job.
+    optimizer_cpu_budget: 0
+```
+
+- If left at 0, Qdrant will keep 1 or more CPUs unallocated - depending on CPU size.
+- If the setting is positive, Qdrant will use this exact number of CPUs for indexing.
+- If the setting is negative, Qdrant will subtract this number of CPUs from the available CPUs for indexing.
+
+For most users, the default `optimizer_cpu_budget` setting will work well. We only recommend you use this if your indexing load is significant.
+
+Our backend leverages dynamic CPU saturation to increase indexing speed. For that reason, the impact on search query performance ends up being minimal. Ultimately, you will be able to strike a the best possible balance between indexing times and search performance.
+
+This configuration can be done at any time, but it requires a restart of Qdrant. Changing it affects both existing and new collections. 
+
+> **Note:** This feature is not available on [Qdrant Cloud](https://qdrant.to/cloud).
+
+## Better indexing for text data
+
+In order to minimize your RAM expenditure, we have developed a new way to index specific types of data. Please keep in mind that this is a backend improvement, and you won't need to configure anything. 
+
+> Going forward, if you are indexing immutable text fields, we estimate a 10% reduction in RAM loads. Our benchmark result is based on a system that uses 64GB of RAM. If you are using less than that, indexing might be faster than 10%.
```

Comment:
> If you are using less than that, indexing might be faster than 10%

is it really about the speed or is it about RAM? The sentence does not sound coherent